docs/cow-protocol/tutorials/widget.md

```diff
@@ -110,23 +110,60 @@ cowSwapWidget(
+| `env`                 | `CowSwapWidgetEnv`     | 'prod'                    | The environment of the widget (`local` , `prod` , `dev` , `pr`). See [`COWSWAP_URLS`](https://github.com/cowprotocol/cowswap/blob/develop/libs/widget-lib/src/consts.ts) const value for urls.                   |
+| `tradeAssets`         | `TradeAssets`          | Same as in swap.cow.fi    | An object containing information about the selling and buying assets. Example: `{ asset: 'WBTC', amount: 12 }` or `{ asset: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48' }`                                      |
+| `theme`               | `CowSwapTheme`         | 'light'                   | The theme of the widget (`'dark'` for dark theme or `'light'` for light theme). It is also possible to set your own colors (`CowSwapWidgetPalette`). See [Custom theme](#custom-theme) section for more details. |
+| `logoUrl`             | `string`               | ---                       | Allows to set a custom logo for the widget.                                                                                                                                                                      |
+| `hideLogo`            | `boolean`              | false                     | Option to hide the logo in the widget.                                                                                                                                                                           |
+| `hideNetworkSelector` | `boolean`              | false                     | Disables an opportunity to change the network from the widget UI.                                                                                                                                                |
+| `enabledTradeTypes`   | `Array<TradeType>`     | All are enabled           | CoW Swap provides three trading widgets: `swap`, `limit` and `advanced` orders. Using this option you can narrow down the list of available trading widgets.                                                     |
+| `partnerFeeBips`      | `string`               | ---                       | Coming soon! You can enable a fee for all trades in the widget. Please contact [us](https://calendly.com/crystal-cow/cow-swap-widget) to enable your partner fee.                                                |
+
+## Custom theme
+
+By setting custom colors you can very flexibly control the appearance of the widget.  
+To do this, you need to pass an object of type `CowSwapWidgetPalette` to the `theme` parameter.
+
+You actually can specify only 2 or 3 basic colors, the rest of the palette is efficiently derived (also with the help of the [color2k](https://github.com/ricokahler/color2k) library to get 'AA' rated text/background color contrasts).
```

Comment:
Maybe point to https://widget.cow.fi so they can experiment

Reply:
There is a link bellow:

>Try it yourself: https://widget.cow.fi